tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/restgenerateurl/GenerateUrl.java

```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.open.jagefilingapi.qa.restgenerateurl;
+
+import ca.bc.gov.open.jagefilingapi.qa.restdata.Payload;
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+
+
+public class GenerateUrl {
+
+   public void generateEfilingUrl() {
+        baseURI = "http://localhost:8080";
+
+        given().log().all().header("Content-Type", "application/json")
+               .body(Payload.submitDocumentProperties()).when().post("submission/generateUrl")
+               .then().log().all().assertThat().statusCode(200).body("efilingUrl", containsString("https://httpbin.org"));
```

Comment:
can we make this more dynamic? like instead of testing that it contains a static string `httpbin.org` which will keep changing, can we make it assert that it has the success navigation url passed in?

Reply:
This is push an initial set up to get the basic rest assured config running. Temporarily removed that assertion. Will be adding along with the other assertion in the next push.